scripts/standard_configure.bat

```diff
@@ -41,16 +52,23 @@ rem set KRATOS_PARALLEL_BUILD_FLAG=/MP4
 
 rem Configure
 @echo on
-cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"          ^
--DUSE_EIGEN_MKL=OFF                                                                                 ^
--DCMAKE_CXX_FLAGS=" %KRATOS_PARALLEL_BUILD_FLAG% "                                                  ^
+cmake                                              ^
+-G "%CMAKE_GENERATOR%"                             ^
+-H"%KRATOS_SOURCE%"                                ^
+-B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"             ^
+-DUSE_EIGEN_MKL=OFF                                ^
+-DCMAKE_CXX_FLAGS=" %KRATOS_PARALLEL_BUILD_FLAG% " ^
 -DKRATOS_GENERATE_PYTHON_STUBS=ON
 
 rem Build
-cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install -- /property:configuration=%KRATOS_BUILD_TYPE% /p:Platform=x64
+IF "%CMAKE_GENERATOR%"=="Ninja" (
+    cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target install -- -j%NUMBER_OF_COMPILATION_CORES%
```

Comment:
```suggestion
    cmake --build "%KRATOS_BUILD%/%KRATOS_BUILD_TYPE%" --target instal -j%NUMBER_OF_COMPILATION_CORES%
```

I'm not sure about windows, but you can pass `-j` directly to CMake on Linux (I think it should work the same way on windows too).

Reply:
As always windows is a little bit special. To this day I still don't know why, but unless set directly in the flags MSVC will ignore the setting.

My guess is that, since there is project target parallel compilation and source level parallel compilation, the -j goes to the project level, so it starts the compilation of several MSVC targets but only 1 source per target.